### PR TITLE
API added parsers for pause/restart/resume actions

### LINKED ISSUE #73: Add Media Action parsing features
This issue is dependent on this issue https://github.com/fossasia/susi_linux/issues/309

The need is the be able to parse resume/restart/pause actions.

### PR DESCRIPTION
**Pull request by sansyrox**
Fixes #73

### CODE REVIEW

**Comment by sansyrox** [comment]
@hongquan @mariobehling @Orbiter @cweitat , please have a look

File: python_wrapper/susi_python/models.py [hunk 1]
```diff
@@ -136,6 +136,11 @@ def __init__(self, identifier , identifier_type):
         self.identifier = identifier
         self.identifier_type = identifier_type
 
+class MediaAction(BaseAction):
+    def __init__(self,mediaAction):
```
Comment (hongquan) [inline python_wrapper/susi_python/models.py]: Coding style

File: python_wrapper/susi_python/main.py [hunk 2]
```diff
@@ -108,6 +108,8 @@ def generate_result(response):
         elif isinstance(action, StopAction):
             result['stop'] = action
             break
+        elif isinstance(action,MediaAction):
```
Comment (hongquan) [inline python_wrapper/susi_python/main.py]: Coding style

**Review by hongquan — CHANGES_REQUESTED** [review]

File: python_wrapper/susi_python/models.py (hunk 1 above)
Comment (sansyrox) [reply python_wrapper/susi_python/models.py]: @hongquan , what is wrong with the coding style?

File: python_wrapper/susi_python/main.py (hunk 2 above)
Comment (sansyrox) [reply python_wrapper/susi_python/main.py]: @hongquan , what is wrong with the coding style?

Comment (hongquan) [reply python_wrapper/susi_python/main.py]: @stealthanthrax https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

Didn't I tell you many times to add plugin to your code editor, to let it check automatically?

**Comment by sansyrox** [comment]
@hongquan , updated

**Review by hongquan — APPROVED** [review]

**Pull request merged**